keyboards/takashicompany/worm_eaten/keymaps/default/keymap.c

```diff
@@ -0,0 +1,85 @@
+// Copyright 2023 takashicompany (@takashicompany)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+    LAYOUT(
```

Comment:
It might be helpful for future users to explicitly list the numbers for each layer.  Eg: 

```suggestion
    [0] = LAYOUT(
```